inference_lib/src/aqlm/inference_kernels/cuda_kernel.cu

```diff
@@ -89,6 +89,87 @@ __global__ void Code1x16MatVec(
   }
 }
 
+
+__global__ void Code1x16Dequant(
+  const int4* __restrict__ A,
+        int4* __restrict__ C,
+  const int4* __restrict__ codebook,
+  int prob_m,
+  int prob_k
+) {
+  int a_gl_stride = prob_k / 8 / 8;
+  int a_gl_rd = (blockDim.x / 32) * blockIdx.x + (threadIdx.x / 32);
+  bool pred = a_gl_rd < prob_m;
+  a_gl_rd = a_gl_stride * a_gl_rd + threadIdx.x % 32;
+  int a_gl_end = a_gl_rd + a_gl_stride - threadIdx.x % 32;
+
+  int c_gl_stride = prob_k / 8;
+  int c_gl_wr = (blockDim.x / 32) * blockIdx.x + (threadIdx.x / 32);
+  c_gl_wr = c_gl_stride * c_gl_wr + (threadIdx.x % 32) * 8;
+
+  int iters = (prob_k / 8 - 1) / (8 * 32) + 1;
+  while (iters--) {
+    int offset = 8 * (threadIdx.x % 32);
```

Comment:
`offset` is never used after being defined here.  Possible bug?  Or just cut and paste error.

Reply:
`offset` used to set `C` offset, but later I realized it's always `a_gl_rd * 8` so I removed it from the memory access, but forgot to remove the initialization and copy-pasted it later. Thanks!

Reply:
removed